frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -0,0 +1,160 @@
+    await clusterSettingsView.resourceLink('console').click();
+    await crudView.isLoaded();
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').getText()).toBe('console');
+
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewSchemaButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.viewSchemaButton.click();
+    await crudView.isLoaded();
+
+  });
+  it('display Global Configuration page, click Configuration Resource item link and display details. Check if the resource link title equals the details page header', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.resourceLink('Console').isDisplayed()).toBe(true);
+    await clusterSettingsView.resourceLink('Console').click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').getText()).toBe('cluster');
+
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewSchemaButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.viewSchemaButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewDetailsButton('metadata').isDisplayed()).toBe(true);
+    await clusterSettingsView.viewDetailsButton('metadata').click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewDetailsButton('initializers').isDisplayed()).toBe(true);
+    await clusterSettingsView.viewDetailsButton('initializers').click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.detailsCloseButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.detailsCloseButton.click();
+    await crudView.isLoaded();
+
+  });
+  it('display Global Configuration page, click dropdown link to edit resource and display details, and check if details header is correct.', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    await clusterSettingsView.globalConfigResourceRow.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.globalConfigKebabDropDownButton('Edit Console Resource').isDisplayed()).toBe(true);
+    await clusterSettingsView.globalConfigKebabDropDownButton('Edit Console Resource').click();
+    await crudView.isLoaded();
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').isDisplayed()).toBe(true);
+    expect(clusterSettingsView.clusterDetailsHeading('resource-title').getText()).toBe('cluster');
+
+    expect(clusterSettingsView.viewSchemaButton.isDisplayed()).toBe(true);
```

Comment:
We should definitely have some tests that cover "View Schema," but I'd suggest moving it into a separate PR. It's not specific to cluster settings.